Sources/TransferMethod/SelectTransferMethodTypeCell.swift

```diff
@@ -86,6 +86,7 @@ extension SelectTransferMethodTypeCell {
                                          Theme.Icon.primaryColor)
             imageView?.image = image
             imageView?.layer.cornerRadius = CGFloat(Theme.Icon.frame.width / 2)
+            detailTextLabel?.accessibilityIdentifier = "\(configuration.transferMethodTypeName ?? "")_value"
```

Comment:
instead of "\(configuration.transferMethodTypeName ?? "")_value" use configuration.transferMethodTypeName

Reply:
fixed